### PR TITLE
Remove messages in chat if they are under a set skyblock level

### PR DESCRIPTION
**Pull request by NopoTheGamer**
non b gon

### CODE REVIEW

**Review by IRONM00N — DISMISSED** [review]

**Review by hannibal002 — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java [hunk 1]
```diff
@@ -53,6 +53,8 @@ public class ChatListener {
 	private final NotEnoughUpdates neu;
 	private static final Pattern SLAYER_XP = Pattern.compile(
 		"   (Spider|Zombie|Wolf|Enderman|Blaze) Slayer LVL (\\d) - (?:Next LVL in ([\\d,]+) XP!|LVL MAXED OUT!)");
+
+	private static final Pattern SKYBLOCK_LVL_MESSAGE = Pattern.compile("\\[(\\d{1,3})\\] .*");
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java:57]: not future proof. what if someone gets sb level 1000

**Review by romangraef — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java (hunk 1 above)
Comment (NopoTheGamer) [reply src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java:57]: /togglechat :+1:

**Review by hannibal002 — DISMISSED** [review]

**Review by IRONM00N — DISMISSED** [review]
still looks good

**Review by romangraef — DISMISSED** [review]
not future proof. what if someone gets sb level 10000

**Comment by IRONM00N** [comment]
@NopoTheGamer fix conflicts

**Comment by NopoTheGamer** [comment]
also ive been told max sb level is 500

**Review by IRONM00N — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Review by romangraef — DISMISSED** [review]

File: Update Notes/2.1.1.md [hunk 2]
```diff
@@ -17,3 +17,4 @@
  - Allow custom delays until skill overlays pause - cobble8
  - Add exponent and percentage to calculator - u9g
  - Add total trophy fish count to /pv - Vixid
+ - Allow to hide messages below a set skyblock level - nopo
```
Comment (IRONM00N) [inline Update Notes/2.1.1.md:20]: Allow hiding

**Review by IRONM00N — CHANGES_REQUESTED** [review]

File: Update Notes/2.1.1.md (hunk 2 above)
Comment (romangraef) [reply Update Notes/2.1.1.md:20]:
```suggestion
 - Allow hiding messages below a set skyblock level - nopo
```

**Review by romangraef — APPROVED** [review]

**Review by walkerselby — APPROVED** [review]

**Pull request merged**